prod/westeurope/internal/appbackend/app_service/terragrunt.hcl

```diff
@@ -221,6 +224,7 @@ inputs = {
 
       // BPD
       ALLOW_BPD_IP_SOURCE_RANGE : "appbackend-ALLOW-BPD-IP-SOURCE-RANGE"
+      JWT_SUPPORT_TOKEN_PRIVATE_RSA_KEY : "appbackend-JWT-SUPPORT-TOKEN-PRIVATE-RSA-KEY"
```

Comment:
I guess we have to create this secret at first, isn't it?

Reply:
@gunzip indeed we have to .... just send me the value on slak and I will do. (cc @AleDore )